capstone/src/main/java/com/google/univiz/scorecard/ScorecardConverter.java

```diff
@@ -0,0 +1,45 @@
+package com.google.univiz.scorecard;
+
+import com.google.common.base.Converter;
+
+/**
+ * ScorecardConverter class returns a representation of ScorecardData as an instance of type
+ * CollegeData.
+ */
+class ScorecardConverter extends Converter<ScorecardData, CollegeData> {
```

Comment:
Can you add a basic unit test to make sure that this logic works? In particular it would be good to verify both paths of the enum parsing logic...